components/Cards/index.js

```diff
@@ -17,3 +17,47 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+axios
+  .get('https://lambda-times-backend.herokuapp.com/articles')
+  .then(response => {
+    for (let key in response.data.articles) {
+      if (response.data.articles.hasOwnProperty(key)) {
+        const articleArray = response.data.articles[key];
+        articleArray.forEach(article => {
+          const cardsContainer = document.querySelector('.cards-container');
+          cardsContainer.append(createArticle(article));
+        });
+      }
+    }
+  })
```

Comment:
Great job using your axios get request! I like how you are console logging the response, and how easy is to follow along your code